.travis.yml

```diff
@@ -0,0 +1,12 @@
+language: node_js
+node_js:
+  - "10.14.1"
```

Comment:
">= 10"?

Reply:
To match package.json

Reply:
Sam had a thing that only worked on node v10.11 for a bit... let's see if that was fixed. I'll try v10

Reply:
using "10" seems to work. I don't think travis supports >= so i'll just go with 10 for now.